sklearn/ensemble/forest.py

```diff
@@ -1633,25 +1633,18 @@ class labels (multi-output problem).
 
     Examples
     --------
-    >>> from sklearn.linear_model import ExtraTreesClassifier
+    >>> from sklearn.ensemble import ExtraTreesClassifier
     >>> from sklearn.datasets import make_classification
 
```

Comment:
Empty lines are rendered as starting a new example in the doc. So I would delete these.

Reply:
@pspachtholz , I have checked the failing of linter and it is failing because at some instance the no of chars in a line exceed the threshold. However, this is not the part of my commit.

Should I fix those too?